apps/web/app/components/explainer.tsx

```diff
@@ -66,11 +70,11 @@ export function ExplainerStep({ index, title, description, img }: EarnInstructio
           {img ?? <Box width="32px" height="32px" />}
         </VStack>
         <styled.h3 mt="space.02" textStyle="label.01">
-          {title}
+          {post ? <PostLabelHoverCard postKey={post.Slug} label={String(title)} /> : title}
```

Comment:
This seems like a really flaky contract - sometimes title is a string but other times it's a `ReactNode`? When is it a string and when is it a node and why?